packages/dashboard/src/components/sidePanel/sections/inputSettingsSection/index.spec.tsx

```diff
@@ -0,0 +1,45 @@
+import { InputWidget } from '../../../../types';
```

Comment:
Suggestion: Use `paths` in this package. https://www.typescriptlang.org/tsconfig#paths

Would let us `import  { DashboardState } from '~/store/state';`

Reply:
Say no more https://github.com/awslabs/iot-app-kit/pull/586